Remove invalid BaseAddress values for ARM64 builds

ARM64 Windows requires base addresses >4GB, but Python 2.7 uses 32-bit addresses.
The solution is to remove BaseAddress specifications for ARM64 and let the linker
choose appropriate addresses automatically.

This patch must be applied AFTER patches 01-04.

--- PCbuild/_bsddb.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/_bsddb.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -68,7 +68,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1e180000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e180000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>

--- PCbuild/_ctypes.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/_ctypes.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -72,7 +72,7 @@
     </ClCompile>
     <Link>
-      <BaseAddress>0x1D1A0000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D1A0000</BaseAddress>
       <AdditionalOptions>/EXPORT:DllGetClassObject,PRIVATE /EXPORT:DllCanUnloadNow,PRIVATE %(AdditionalOptions)</AdditionalOptions>
     </Link>
   </ItemDefinitionGroup>

--- PCbuild/_elementtree.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/_elementtree.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -69,7 +69,7 @@
       <PreprocessorDefinitions>_CRT_SECURE_NO_DEPRECATE;_CRT_NONSTDC_NO_DEPRECATE;USE_PYEXPAT_CAPI;XML_STATIC;HAVE_EXPAT_CONFIG_H;XML_POOR_ENTROPY;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
-      <BaseAddress>0x1D100000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D100000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>

--- PCbuild/_msi.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/_msi.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -66,7 +66,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>fci.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1D160000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D160000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>

--- PCbuild/_multiprocessing.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/_multiprocessing.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -66,7 +66,7 @@
       <PreprocessorDefinitions>Py_BUILD_CORE_MODULE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
-      <BaseAddress>0x1e1D0000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e1D0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>

--- PCbuild/_socket.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/_socket.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -66,7 +66,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1e1D0000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e1D0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>

--- PCbuild/_sqlite3.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/_sqlite3.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -68,7 +68,7 @@
       <PreprocessorDefinitions>MODULE_NAME="sqlite3";%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
-      <BaseAddress>0x1e180000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e180000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>

--- PCbuild/_testcapi.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/_testcapi.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -66,7 +66,7 @@
       <PreprocessorDefinitions>Py_BUILD_CORE_MODULE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
-      <BaseAddress>0x1e1F0000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e1F0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>

--- PCbuild/bz2.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/bz2.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -68,7 +68,7 @@
       <PreprocessorDefinitions>_CRT_SECURE_NO_DEPRECATE;_CRT_NONSTDC_NO_DEPRECATE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
-      <BaseAddress>0x1D170000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D170000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>

--- PCbuild/python.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/python.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -72,7 +72,7 @@
     </ResourceCompile>
     <Link>
       <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1d000000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1d000000</BaseAddress>
       <SubSystem>Console</SubSystem>
     </Link>
   </ItemDefinitionGroup>

--- PCbuild/pythoncore.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/pythoncore.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -74,7 +74,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1e000000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1e000000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>

--- PCbuild/pythonw.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/pythonw.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -72,7 +72,7 @@
     </ResourceCompile>
     <Link>
       <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1d000000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1d000000</BaseAddress>
       <SubSystem>Windows</SubSystem>
     </Link>
   </ItemDefinitionGroup>

--- PCbuild/select.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/select.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -66,7 +66,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1D110000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D110000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>

--- PCbuild/unicodedata.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/unicodedata.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -66,7 +66,7 @@
       <PreprocessorDefinitions>Py_BUILD_CORE_MODULE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
-      <BaseAddress>0x1D120000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D120000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>

--- PCbuild/winsound.vcxproj.orig	2025-12-04 15:17:00.846057900 +0000
+++ PCbuild/winsound.vcxproj	2025-12-04 15:17:19.009930900 +0000
@@ -66,7 +66,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>winmm.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <BaseAddress>0x1D160000</BaseAddress>
+      <BaseAddress Condition="'$(Platform)' != 'ARM64'">0x1D160000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
